Automattic/newspack-theme
# Chore: Update theme POT file and translation files

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the theme's POT files, and the existing German, Spanish and Brazilian Portuguese translations.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Rotate through the three languages -- German, Spanish and Brazilian Portuguese -- and confirm that translations are still appearing on the front-end, and in the editor, where they exist (a good spot to check is the Featured Image sizes, since Large is always translated).
3. Review the diff and confirm that no existing translations have been changed, and only line numbers, new strings have been added, with the following exceptions:
   * The footer credit has been changed and translation has been updated (originally just credited WordPress and linked to .org; now credits Newspack and Automattic, and links to https://newspack.pub).
   * The theme now has some of the original WooCommerce checkout strings in it; they should also include translations.
   * Some translation comments were also cleaned up, so the ' and ' and ', ' translations moved (they used to just be used for multiple authors, but now they are also used for multiple sponsors). 

### Other information:
